src/hooks/useDisableMenuOnOutsideClick.js

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+export default (ref, setNavActive) => {
+  useEffect(() => {
+    function handleClickOutside(event) {
```

Comment:
why not arrow function?

nice try - did you find this on internet? 
Anyway, I would suggest to find some package for handling outside click.

And... is this hook doesn't make anything as disabled, so name is not valid.